Source/Core/DolphinQt/ConvertDialog.cpp

```diff
@@ -149,7 +149,7 @@ void ConvertDialog::OnFormatChanged()
   case DiscIO::BlobType::GCZ:
   {
     // To support legacy versions of dolphin, we have to check the GCZ block size
-    // See DiscIO::IsGCZBlockSizeLegacyCompatible() in DiscUtil.cpp for details
+    // See IsGCZBlockSizeLegacyCompatible() for details
```

Comment:
Nitpick (feel free to ignore): The `DiscIO::` prefix does make sense to include.